modules/terragrunt/init.go

```diff
@@ -22,8 +18,8 @@ func TgStackInit(t testing.TestingT, options *Options) string {
 
 // TgStackInitE calls terragrunt init and return stdout/stderr
 func TgStackInitE(t testing.TestingT, options *Options) (string, error) {
```

Comment:
I think the naming convention for files should be updated - this file is not for `terragrunt init` but for `terragrunt stack init`
The file name most probably should be `stack_init.go`  (and other files too)